src/Microsoft.NetCore.Analyzers/UnitTests/Performance/MutableStructsShouldNotBeUsedForReadonlyFieldsTests.Fixer.cs

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.NetCore.CSharp.Analyzers.Performance;
+using Microsoft.NetCore.VisualBasic.Analyzers.Performance;
+using Test.Utilities;
+using Xunit;
+
+namespace Microsoft.NetCore.Analyzers.Performance.UnitTests
+{
+    public class MutableStructsShouldNotBeUsedForReadonlyFieldsFixerTests
+    {
+        [Fact]
+        public async Task CSharpReadonlyFieldsOfKnownMutableTypes_RemovesReadonlyModifier()
+        {
+            var initial = @"
+using System;
+using System.Threading;
+using System.Runtime.InteropServices;
+
+public class C
+{
+    private [|readonly|] SpinLock _sl = new SpinLock();
+    private [|readonly|] GCHandle _gch = new GCHandle();
+
+    private [|readonly|] SpinLock _sl_noinit;
+    private [|readonly|] GCHandle _gch_noinit;
+}
+";
+
+            var expected = @"
+using System;
+using System.Threading;
+using System.Runtime.InteropServices;
+
+public class C
+{
+    private SpinLock _sl = new SpinLock();
+    private GCHandle _gch = new GCHandle();
+
+    private SpinLock _sl_noinit;
+    private GCHandle _gch_noinit;
+}
+";
+            await CSharpCodeFixVerifier<CSharpMutableStructsShouldNotBeUsedForReadonlyFieldsAnalyzer, CSharpMutableStructsShouldNotBeUsedForReadonlyFieldsFixer>
```

Comment:
Kindly follow the pattern used by all the existing unit tests in the repo and define usings to avoid clutter in individial unit tests such as https://github.com/dotnet/roslyn-analyzers/blob/c4e547383037fc989178ad61974b3de5547ffb76/src/Microsoft.NetCore.Analyzers/UnitTests/Runtime/TestForNaNCorrectlyTests.Fixer.cs#L8-L52